### PR TITLE
chore(docs): add initial migration documentation for v1 to v2

### PR DESCRIPTION
**Pull request by janechu**
# Pull Request

## 📖 Description

This adds initial documentation for the site for v1 to v2 of the packages `@microsoft/fast-element`, `@microsoft/fast-foundation` and `@microsoft/fast-components`. This assumes that all three packages will major version bump.

## 👩‍💻 Reviewer Notes

Please check the wording carefully, this documentation is based on my recent experience with migrating the experimental web components in the `@microsoft/fast-tooling` package.

Reviewers may want to build the docusaurus site to double check the page appears as it should.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by chrisdholt — APPROVED** [review]

File: sites/website/src/docs/migrate/1-to-2.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Migrating from v1 to v2
+
+This guide aims to help you migrate from `@microsoft/fast-element`, `@microsoft/fast-foundation`, and `@microsoft/fast-components` v1 to v2.
```
Comment (EisenbergEffect) [inline sites/website/src/docs/migrate/1-to-2.md:3]: No changes to `@microsoft/fast-element` so that can be removed from here.

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]
One minor change.
QQ: Do we want to add this to the site ToC? If so, where should we put it?

**Review by nicholasrice — APPROVED** [review]
Thanks Jane! I'll do some iterations on this once we get it in.

**Comment by bheston** [comment]
This is a good start. I see Nick said he would add to it. For instance we're missing a note about migrating color recipes from behaviors.

I think the descriptions in each section are helpful, but I wonder if we should also have a quick checklist like
- Update components to extend FoundationElement
- Replace template definition from a to b
- Replace style definition...
- Search and replace `behavior.var`
- Remove color behavior imports and withBehaviors
- etc

Those might be the bulk of the steps, tersely written obviously.

**Comment by janechu** [comment]
Closing this as the migration documents are being re-written within an updated docs site structure.

**Pull request closed without merging**